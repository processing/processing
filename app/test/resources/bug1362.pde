if (true) {} else { new String(); }