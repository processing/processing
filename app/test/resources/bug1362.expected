import processing.core.*; 

import java.applet.*; 
import java.awt.Dimension; 
import java.awt.Frame; 
import java.awt.event.MouseEvent; 
import java.awt.event.KeyEvent; 
import java.awt.event.FocusEvent; 
import java.awt.Image; 
import java.io.*; 
import java.net.*; 
import java.text.*; 
import java.util.*; 
import java.util.zip.*; 
import java.util.regex.*; 

public class bug1362 extends PApplet {
    public void setup() {
if (true) {} else { new String(); }
    noLoop();
} 
    static public void main(String args[]) {
        PApplet.main(new String[] { "--bgcolor=null", "bug1362" });
    }
}
